Guides/Getting Started.md

```diff
@@ -1,3 +1,128 @@
+
+```swift
+let newTableViewModel: TableViewModel = ...
+self.tableViewDriver?.tableViewModel = newTableViewModel
+```
+
+Great!  But how do you make that `TableViewModel`?  We recommend having a static function that
+takes in new data and generates the `TableViewModel`.  It might look something like this:
+
+```swift
+static func viewModel(forState people: [Person]) -> TableViewModel { }
+
+```
+
+Then any time your data (in this case the `people` property), you can generate your new `TableViewModel`
+
+```swift
+var people: [Person] = [] {
+    didSet {
+        self.tableViewDriver?.tableViewModel = PersonViewController.viewModel(
+            forState: people
+        )
+    }
+}
+
+```
+
+Okay now lets go back and fill in our `viewModel()` function:
+
+```swift
+let sections: [TableViewSectionViewModel] = people.map { person in
+    let personCellViewModels = people.tools.map { PersonCellModel(tool: $0) }
+    return TableViewSectionViewModel(
+        headerTitle: "People",
+        headerHeight: 44,
+        cellViewModels: personCellViewModels,
+        diffingKey: "People" // a unique string for automatically diffing
+    )
+}
+return TableViewModel(sectionModels: sections)
+```
+
+This is now called whenever we get an updated to our `people` variable.  This function takes the Latest
+`people` data and for each person generates a `PersonCellModel` to display in our table view.  It wraps
+all those models into a single section and then creates a `TableViewModel` from that section.
+
+Now all we have to do is to define `PersonCellModel`:
+
+```swift
+struct PersonCellModel: TableViewCellViewModel, DiffableViewModel {
+    var accessibilityFormat: CellAccessibilityFormat = "PersonUserCell"
+    let cellIdentifier = "PersonUserCell"
+    let commitEditingStyle: CommitEditingStyleClosure?
+    let editingStyle: UITableViewCellEditingStyle = .delete
+    let person: Person
+
+    init(person: Person) {
+        self.person = person
+    }
+
+    func applyViewModelToCell(_ cell: UITableViewCell) {
+        cell.textLabel?.text = "\(self.person.name)""
+    }
+
+    // If we want the library to automatically reload when new data is available
+    // each cell model needs to have a unique key for the diffing algorithm to work.
+    // In this case we just use the person's uuid
+    var diffingKey: String {
+        return self.person.uuid.uuidString
+    }
+}
+```
+
+A `CellViewModel` just needs to conform to `TableViewCellViewModel` or `CollectionViewCellViewModel`.  The
+most important function is `applyViewModelToCell`.  This is where you define how your view model will get
+mapped onto a `UITableViewCell` or `UICollectionViewCell`.  In this case we are just setting the `textLabel`'s
+`text` property to the person's `name`.
+
+And there you have it!  You now have an automatically refreshing table view which you defined in a clear
+declarative manner.
+
+
+### Glossary of types
```

Comment:
RE: comment above

I think this should read "Primary Components", then have the explanations below. 😊